main.yml

```diff
@@ -447,18 +447,20 @@ inventory:
 
 image-builder:
   title: Image Builder
+  api:
+    versions:
+      - v1
   deployment_repo: https://github.com/RedHatInsights/image-builder-frontend-build
-  permissions:
```

Comment:
image-builder isn't in the nav yet for prod-stable. I wanted to wait with that since our GA date is may 2nd (thought of doing it like 28th of april). Unless we can just go ahead and add it now?